.cargo/config.toml

```diff
@@ -2,15 +2,25 @@
 CARGO_WORKSPACE_DIR = { value = "", relative = true }
 
 [build]
-
+rustflags = [
+  "--cfg",
+  "tokio_unstable",
+  "-Zshare-generics=y", # make the current crate share its generic instantiations
+  "-Zthreads=8", # parallel frontend https://blog.rust-lang.org/2023/11/09/parallel-rustc.html
```

Comment:
Nice!

Given that this is still pretty experimental and has known bugs, I think it'd be better to do this in a separate commit/PR so that if we run into issues and have to bisect, it's easier to isolate.

The blog post says:

> In multi-threaded mode there are some known bugs, including deadlocks. If compilation hangs, you have probably hit one of them.